tasks/tag-and-push.yml

```diff
@@ -13,11 +13,7 @@
     force: true
 
 - name: Apply additional tag if there's more than one.
-  docker_image:
-    name: "{{ ecr_image_name }}:{{ ecr_image_tags.0 }}"
-    repository: "{{ ecr_image_name }}:{{ image_tag }}"
-    state: present
-    pull: false
+  command: "docker tag {{ ecr_image_name }}:{{ ecr_image_tags.0 }} {{ ecr_url }}/{{ ecr_image_name }}:{{ image_tag }}"
   when: ecr_image_tags.0 != image_tag
```

Comment:
Can you also remove this `when` condition?

We will no longer need it since even that tag needs to be tagged with the `ecr_url` as well.